ui/components-react/src/components-react/filter-builder/FilterBuilderToolbar.tsx

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module PropertyFilterBuilder
+ */
+import React from "react";
+import { Flex, IconButton } from "@itwin/itwinui-react";
+import { SvgAdd, SvgDelete } from "@itwin/itwinui-icons-react";
+import { UiComponents } from "../UiComponents";
+import { getiTwinUISize } from "../common/iuiUtils";
+/**
+ * Props for [[PropertyFilterBuilderToolbar]] component.
+ * @internal
+ */
+interface PropertyFilterBuilderToolbarProps {
+  /** Size to render the component. If undefined, defaults to iTwinUI "small" size. */
+  size?: "medium" | "large";
+  /** Function to add child of current group. */
+  onAddChild: () => void;
+  /** Function to remove child from current group. */
+  onDelete: () => void;
+}
+
+/** Toolbar displaying the "add" and "delete" row buttons in the filter builder
+ * @beta
+ */
+export const PropertyFilterBuilderToolbar = (
+  props: PropertyFilterBuilderToolbarProps
+) => {
+  const { size, onAddChild, onDelete } = props;
+
+  return (
+    <Flex
+      className={"fb-toolbar fb-row-toolbar"}
+      gap="0"
+      justifyContent="flex-end"
+    >
+      <IconButton
+        size={getiTwinUISize(size)}
+        className="fb-add-rule-button"
+        data-testid="fb-add-rule-button"
```

Comment:
I think `data-testid` can be removed because buttons can be selected by class name.